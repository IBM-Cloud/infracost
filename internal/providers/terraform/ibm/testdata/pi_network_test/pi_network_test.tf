
terraform {
  required_providers {
    ibm = {
      source  = "IBM-Cloud/ibm"
      version = "~> 1.40.0"
    }
  }
}

provider "ibm" {
  region = "us-south"
  zone   = "dal12"
}

locals {
  service_type = "power-iaas"
}

resource "ibm_resource_group" "resource_group" {
  name = "default"
}

resource "ibm_resource_instance" "powervs_service" {
  name              = "Power instance"
  service           = local.service_type
  plan              = "power-virtual-server-group"
  location          = "us-south"
  resource_group_id = ibm_resource_group.resource_group.id
}

resource "ibm_pi_network" "public_vlan_power_network" {
  count                = 1
  pi_network_name      = "power-network"
  pi_cloud_instance_id = ibm_resource_instance.powervs_service.guid
  pi_network_type      = "pub-vlan"
}
