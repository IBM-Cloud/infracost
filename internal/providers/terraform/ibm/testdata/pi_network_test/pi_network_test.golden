
 Name                                         Monthly Qty  Unit      Monthly Cost 
                                                                                  
 ibm_pi_network.public_vlan_power_network[0]                                      
 └─ power-network                                       1  Instance         $0.00 
                                                                                  
 ibm_resource_instance.powervs_service                                            
 └─ Power instance                                      1  Instance         $0.00 
                                                                                  
 OVERALL TOTAL                                                              $0.00 
──────────────────────────────────
3 cloud resources were detected:
∙ 2 were estimated, 1 of which usage-based costs, see https://infracost.io/usage-file
∙ 1 was free:
  ∙ 1 x ibm_resource_group